stubs/default/routes/web.php

```diff
@@ -22,10 +22,10 @@
     return view('dashboard');
 })->middleware(['auth', 'verified'])->name('dashboard');
 
-Route::middleware('auth')->group(function () {
-    Route::get('/profile', [ProfileController::class, 'edit'])->name('profile.edit');
-    Route::patch('/profile', [ProfileController::class, 'update'])->name('profile.update');
-    Route::delete('/profile', [ProfileController::class, 'destroy'])->name('profile.destroy');
+Route::controller(ProfileController::class)->middleware(['auth', 'verified'])->group(function () {
```

Comment:
Adding the verified method here, is not needed.

Reply:
I should not using verified here.